ui-components/src/locales/general.json

```diff
@@ -1372,6 +1372,7 @@
     "seeMoreOpportunities": "See more rental and ownership housing opportunities",
     "checkEligibility": "Do I Qualify?",
     "checkEligibilityDescription": "Check in Minutes if You Qualify!",
+    "checkQualifications": "Check Your Housing Qualifications",
```

Comment:
The text seems very long but it is what the ticket asked for. How does this look in mobile?

Reply:
It doesn't look super great to me, but it doesn't appear to be causing any issues:

![Screenshot from 2021-11-04 17-01-22](https://user-images.githubusercontent.com/66751489/140436640-17f0577e-8f95-4e36-a4c1-d23e5a2749ce.png)